js/plugin/connection.ts

```diff
@@ -66,32 +86,38 @@ export const ConnectionMixin = (SuperClass) => {
       const conn = (await hass()).connection;
       this.connection = conn;
 
-      // Subscribe to configuration updates
-      conn.subscribeMessage((msg) => this.incoming_message(msg), {
-        type: "browser_mod/connect",
-        browserID: this.browserID,
-      });
+      const setupBrowserModConnection = () => {
+        this.LOG("Subscribing to browser_mod/connect events");
+        conn.subscribeMessage((msg) => this.incoming_message(msg), {
+          type: "browser_mod/connect",
+          browserID: this.browserID,
+        });
+      };
+
+      // Initial connect subscription
+      setupBrowserModConnection();
+      // If this fails, then:
+      // Observe `component_loaded` to track when `browser_mod` is dynamically added
+      conn.subscribeEvents((event) => {
+        if (event.data?.component === "browser_mod") {
+          this.LOG("Detected browser_mod component load");
+          setupBrowserModConnection();
+        }
+      }, "component_loaded");
```

Comment:
This is the clue.

This will never be triggered if HA was already up when a browser was opened.